addons/editor-devtools/DevTools.js

```diff
@@ -1,6 +1,5 @@
 // import ShowBroadcast from "./show-broadcast.js";
 import DomHelpers from "./DomHelpers.js";
-import XML from "./XML.js";
 import UndoGroup from "./UndoGroup.js";
 
 export default class DevTools {
```

Comment:
Can't put a review comment on it because you haven't changed it but you can remove the `floatInp` and `blockCursor` vars from the constructor of `DevTools`